lib/modules/manager/flux/common.ts

```diff
@@ -1,7 +1,7 @@
 import { regEx } from '../../../util/regex';
 
 export const systemManifestRegex =
-  '(^|\\/)flux-system\\/(?:.+\\/)?gotk-components\\.yaml$';
+  '(^|/)flux-system\\/(?:.+\\/)?gotk-components\\.yaml$';
```

Comment:
```suggestion
  '(^|/)flux-system/(?:.+/)?gotk-components\\.yaml$';
```
should remove all escapes